pkg/kubelet/kubelet_node_status.go

```diff
@@ -606,8 +606,16 @@ func (kl *Kubelet) setNodeStatusMachineInfo(node *v1.Node) {
 		}
 
 		for _, removedResource := range removedDevicePlugins {
-			glog.V(2).Infof("Remove capacity for %s", removedResource)
-			delete(node.Status.Capacity, v1.ResourceName(removedResource))
+			glog.V(2).Infof("Set capacity for %s to 0 on device removal", removedResource)
```

Comment:
@jiayingz, please take a look.

Reply:
should allocatable also be set to 0 ?

Reply:
Allocatable will be set to capacity at line 644.